shell/platform/android/io/flutter/plugin/platform/PlatformViewWrapper.java

```diff
@@ -202,19 +238,32 @@ public void draw(Canvas canvas) {
       Log.e(TAG, "Invalid texture. The platform view cannot be displayed.");
       return;
     }
-    // Override the canvas that this subtree of views will use to draw.
-    final Canvas surfaceCanvas = surface.lockHardwareCanvas();
-    try {
-      // Clear the current pixels in the canvas.
-      // This helps when a WebView renders an HTML document with transparent background.
-      if (Build.VERSION.SDK_INT >= 29) {
-        surfaceCanvas.drawColor(Color.TRANSPARENT, BlendMode.CLEAR);
-      } else {
-        surfaceCanvas.drawColor(Color.TRANSPARENT);
+    // We've observed on Android Q that we have to wait for the consumer of {@link SurfaceTexture}
```

Comment:
did you test with versions below Q?

Reply:
cc @jreck

Reply:
> did you test with versions below Q?

Yes, I did. They work fine.

Reply:
Just to clarify, which versions did you test on? I'm getting reports that it may be several versions. See https://github.com/googleads/googleads-mobile-flutter/issues/269#issuecomment-1059832373

Reply:
> Just to clarify, which versions did you test on? I'm getting reports that it may be several versions. See [googleads/googleads-mobile-flutter#269 (comment)](https://github.com/googleads/googleads-mobile-flutter/issues/269#issuecomment-1059832373)

I have tested on real devices from Android9 to Android12.